cheatsheets/PHP_Configuration_Cheat_Sheet.md

```diff
@@ -107,16 +104,10 @@ memory_limit            = 50M
 post_max_size           = 20M
 max_execution_time      = 60
 report_memleaks         = On
-track_errors            = Off
 html_errors             = Off
+zend.exception_ignore_args = Off
```

Comment:
```suggestion
zend.exception_ignore_args = On
```
0/Off (disabled) is the default value. 1/On (enabled) means arguments will be excluded from stack traces.
https://www.php.net/manual/en/ini.core.php#ini.zend.exception-ignore-args